pixaxe/pixaxe.py

```diff
@@ -194,169 +169,55 @@ def find_additional_content(self, alfile):
-            if self.config['run_steg_auto'] or run_steg:
-                decloak = True
-            else:
-                decloak = False
-            try:
-                imginfo = ImageInfo(infile, request, result, self.working_directory, self.log)
-            except NotSupported:
-                decloak = False
-
-            # Run Tesseract on sample
-            # Process the command and save the csv result in the result object
-            usable_out = None
-            orig_outfile = os.path.join(self.working_directory, 'outfile')
-            stdout, stderr = self.tesseract_call(infile, orig_outfile)
-
-            if stdout or stderr:
-                # Assess Tesseract warnings
-                if b"pix too small" in stderr:
-                    # Make the image larger with convert command
-                    c_outfile = os.path.join(self.working_directory, 'enlrg_img')
-                    c_stdout, c_stderr = self.convert_img(infile, c_outfile)
-                    if c_stdout:
-                        c_outfile = os.path.join(self.working_directory, 'c_outfile')
-                        enlrg_infile = os.path.join(self.working_directory, 'enlrg')
-                        if not c_stderr:
-                            stdout, stderr = self.tesseract_call(enlrg_infile, c_outfile)
-                            if stdout:
-                                if not stderr:
-                                    outfile = c_outfile
-                                else:
-                                    outfile = orig_outfile
-                            else:
-                                outfile = orig_outfile
-                        else:
-                            outfile = orig_outfile
-                    else:
-                        outfile = orig_outfile
-                else:
-                    outfile = orig_outfile
-                    self.log.debug("Tesseract errored/warned on sample {}. Error:{}".format(self.sha, stderr))
 
-                usable_out = self.assess_output(outfile, request)
-
-            if usable_out:
-                ores = ResultSection("OCR Engine detected strings in image",
-                                     body_format=BODY_FORMAT.MEMORY_DUMP)
-                ores.add_line("Text preview (up to 500 bytes):\n")
-                ores.add_line("{}".format(usable_out[0:500]))
-                result.add_section(ores)
+        # Always provide a preview of the image being analyzed
+        image_preview = ResultImageSection(request, "Image Preview")
+        image_preview.add_image(request.file_path, name=request.file_name, description='Input file')
+        result.add_section(image_preview)
+
+        # Look for suspicious OCR
+        detections = ocr_detections(request.file_path)
+        if detections:
+            self.log.info('OCR detections found.')
+            result.add_section(ResultSection(
```

Comment:
since we're breathing new life, use the ResultKeyValueSection
https://github.com/CybercentreCanada/assemblyline-v4-service/blob/60ee2a7b5c7fe6021913d1e2889677dc83c43856/assemblyline_v4_service/common/result.py#L614